concepticondata/conceptlists/Gauchat-1925-480.tsv

```diff
@@ -0,0 +1,481 @@
+Gauchat-1925-480-1	1	It is	Il fait	facit	2		
+Gauchat-1925-480-2	2	hot	chaud	caldu	2	1286	HOT
+Gauchat-1925-480-3	3	today	aujourd‘hui	hŏdie	2	1283	TODAY
+Gauchat-1925-480-4	4	This is	C’est	est	2		
+Gauchat-1925-480-5	5	a good	un bon	bŏnu	2	1035	GOOD
+Gauchat-1925-480-6	6	weather	temps	tĕmpus	2	952	WEATHER
+Gauchat-1925-480-7	7	A (feminine)	Une	ūna	4		
+Gauchat-1925-480-8	8	good (feminine)	bonne	bŏna	4		
+Gauchat-1925-480-9	9	season	saison	satiōne	4	691	SEASON
+Gauchat-1925-480-10	10	It rains	Il pleut	plŏvet	4		
+Gauchat-1925-480-11	11	each	chaque	cata + quisque	4	3416	EVERY (EACH)
+Gauchat-1925-480-12	12	day	jour	diŭrnu	4		DAY (NOT NIGHT)
+Gauchat-1925-480-13	13	yesterday	hier	hĕri	6	1174	YESTERDAY
+Gauchat-1925-480-14	14	there was	il y avait	habēbat	6		
+Gauchat-1925-480-15	15	little	peu de	paucu	6	1242	FEW
+Gauchat-1925-480-16	16	dew	rosée	rosata	6	1977	DEW
+Gauchat-1925-480-17	17	It was	Il faisait	faciēbat	6		
+Gauchat-1925-480-18	18	cold	froid	frĭgidu	6	1287	COLD
+Gauchat-1925-480-19	19	The morning	La matinée	matutinata	8	1339	MORNING
+Gauchat-1925-480-20	20	was	était	ĕrat (ĕst + ēbat)	8		
+Gauchat-1925-480-21	21	cold (feminine)	froide	frĭgida	8		
+Gauchat-1925-480-22	22	The mornings	Les matinées	matutinatas	8		
+Gauchat-1925-480-23	23	are	sont	sŭnt	8		
+Gauchat-1925-480-24	24	cold (feminine plural)	froides	frĭgidas	8		
+Gauchat-1925-480-25	25	The frost	La gelée	gelata	10	2034	FROST
+Gauchat-1925-480-26	26	is dangerous (feminine)	est dangereuse	dominiariu + ōsas	10		
+Gauchat-1925-480-27	27	The frosts	Les gelées	gelatas	10		
+Gauchat-1925-480-28	28	are dangerous	sont dangereuses	dominiariu + ōsas	10		
+Gauchat-1925-480-29	29	The night	La nuit	nŏcte	10	1233	NIGHT
+Gauchat-1925-480-30	30	one sees	on voit	hŏmo vĭdet	10		
+Gauchat-1925-480-31	31	the moon	la lune	lūna	12	1313	MOON
+Gauchat-1925-480-32	32	the stars	les étoiles	stēllas	12	1430	STAR
+Gauchat-1925-480-33	33	One does not have	Il ne faut pas	fallit	12		
+Gauchat-1925-480-34	34	to have	avoir	habēre	12	773	HAVE
+Gauchat-1925-480-35	35	fear	peur	pavōre	12	781	FEAR (FRIGHT)
+Gauchat-1925-480-36	36	from the thunder	du tonnerre	tonĭtru	12	1150	THUNDER
+Gauchat-1925-480-37	37	In the month	Au mois	mēse	14		
+Gauchat-1925-480-38	38	of February	de février	febrariu	14	2037	FEBRUARY
+Gauchat-1925-480-39	39	there is	il y a	habet	14	2152	THERE IS OR THERE ARE
+Gauchat-1925-480-40	40	snow	de la neige	nĭve	14	784	SNOW
+Gauchat-1925-480-41	41	ice	de la glace	glacia	14	617	ICE
+Gauchat-1925-480-42	42	It is (2)	Il fait	facit	14		
+Gauchat-1925-480-43	43	some wind	du vent	vĕntu (aura)	16	960	WIND
+Gauchat-1925-480-44	44	Winter	L’hiver	hibĕrnu	16	1471	WINTER
+Gauchat-1925-480-45	45	arrives	arrive	adrīpat	16	1387	ARRIVE
+Gauchat-1925-480-46	46	It is there	Il est là	illac	16		
+Gauchat-1925-480-47	47	I must	Je dois	debēo	16		
+Gauchat-1925-480-48	48	go down	descendre	descĕndere	16	2014	GO DOWN (DESCEND)
+Gauchat-1925-480-49	49	in the city	en ville	vīlla	18		
```

Comment:
Question: If function words should generally be ignored, why can't this be CITY then? "from the thunder" above is mapped to THUNDER.

Reply:
Yes, it can be mapped. However, I think now, we should not get too much into detail and later make an updated list where we map the concrete words in inter-linear glosses of this text. So whether we map "in the city" now, or not, is not that important.